### ISSUE TITLE
Unable to run jar file via terminal

### ISSUE THREAD

**Issue by tanteckfang** [nus-cs2103-AY2324S1/forum#385]
I'm currently operating on an M2 Mac and have installed the Java version suggested for this module. Nevertheless, every attempt to execute the jar file via the terminal results in an error. The jar file runs when I initiate it by right-clicking and selecting JavaLauncher, I'm concerned that this way might not be ideal, as it results in a blank address book. I would be truly grateful for any advice on how to resolve this issue.
<img width="908" alt="Screenshot 2023-11-09 at 11 22 34 PM" src="https://github.com/nus-cs2103-AY2324S1/forum/assets/110968724/2269ac39-6604-4f72-9263-6db0c8f828ee">

my java version
<img width="897" alt="Screenshot 2023-11-09 at 11 30 15 PM" src="https://github.com/nus-cs2103-AY2324S1/forum/assets/110968724/f9662ac5-e62c-4e76-bb75-0d54852932c8">

**Comment by damithc** [comment]
@tanteckfang Do you have any other JDK versions installed in your computer?

**Comment by tanteckfang** [comment]
No Prof, I checked my java version in my computer and I only have this 
<img width="569" alt="Screenshot 2023-11-09 at 11 53 27 PM" src="https://github.com/nus-cs2103-AY2324S1/forum/assets/110968724/33a96d9d-3d89-484f-beb8-cd12702ddc1a">

**Comment by damithc** [comment]
Which JAR file are you trying? Was it created in your own computer?
Are you able to run the [original AB3 JAR file](https://github.com/se-edu/addressbook-level3/releases)?

**Comment by tanteckfang** [comment]
Unfortunately no prof, I can't run the original AB3 JAR file
<img width="766" alt="Screenshot 2023-11-10 at 12 11 43 AM" src="https://github.com/nus-cs2103-AY2324S1/forum/assets/110968724/b6a02fdc-d015-448e-bbd6-33d058be7b3f">
, but I can run the zip addressbook you shared here https://github.com/nus-cs2103-AY2324S1/forum/issues/225#issuecomment-1739713892

I am able to run my group's JAR file by editing the build.gradle by adding ` id 'org.openjfx.javafxplugin' version '0.0.13'` at plugins and `javafx {
    version = '17'
    modules = [ 'javafx.controls','javafx.graphics', 'javafx.fxml' ]
}` 
<img width="863" alt="Screenshot 2023-11-10 at 12 16 29 AM" src="https://github.com/nus-cs2103-AY2324S1/forum/assets/110968724/db0c29e5-a3e2-4769-bb53-da669b04b9ba">
And I understand that I could not edit other group's build.gradle during PE.

**Comment by damithc** [comment]
> Unfortunately no prof, I can't run the original AB3 JAR file

I see. In that case this is serious and need to be fixed before the PE.
See if https://github.com/nus-cs2103-AY2324S1/forum/issues/225#issuecomment-1744945712 helps

**Comment by tanteckfang** [comment]
Hi Prof, I have tried the suggested way for that issue but it does not work for me, however, I tried to download javafx onto my computer and use this command `java --module-path /Users/tanteckfang/Documents/javafx-sdk-17.0.9/lib --add-modules javafx.controls,javafx.fxml,javafx.graphics -jar addressbook.jar`, it works fine to me now. 
Thank you very much for your suggestions and the prompt reply. I appreciate your guidance on this matter.
